### PR TITLE
restore: set configed keepalive for restore

### PR DESCRIPTION
**Pull request by YuJuncen**
### What problem does this PR solve? 
#545 added the config value of keepalive, but it has forgotten the restore part(!).

### What is changed and how it works?
Apply the config to restoring.

### Check List 

Tests 

 - Unit test
 - Integration test
(If the old tests passed, then the default config should be applied.)

### Release Note

 - Fix a bug that causes the keepalive configs don't apply to restore.

### CODE REVIEW

File: pkg/task/restore.go [hunk 1]
```diff
@@ -102,7 +102,9 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 	}
 	defer mgr.Close()
 
-	client, err := restore.NewRestoreClient(g, mgr.GetPDClient(), mgr.GetTiKV(), mgr.GetTLSConfig())
+	keepaliveCfg := GetKeepalive(&cfg.Config)
+	keepaliveCfg.PermitWithoutStream = true
```
Comment (3pointer) [inline pkg/task/restore.go:106]: How about put this line in GetKeepalive function?

Comment (YuJuncen) [reply pkg/task/restore.go:106]: This function is also used in the backup site, which doesn't needs `PermitWithoutStream`.

**Review by 3pointer — COMMENTED** [review]
LGTM

**Review by kennytm — APPROVED** [review]
LGTM

**Pull request merged**